ads/js/sticky.js

```diff
@@ -1,111 +1,125 @@
 const debounce = require('./utils').debounce;
-const stickyNavHeight = 74; // for use with Right Hand Rail
 
 function Sticky (el, opts) {
-	if (!el) return;
-	this.el = el;
 	this.opts = opts || {};
+	this.fixed = el;
+	this.boundary = document.querySelector(this.opts.stickUntil);
+	this.eventdbScrollEnd = debounce(this.scrollEnd.bind(this), 300);
+	this.eventScrollStart = this.scrollStart.bind(this);
 	this.sibling = opts.sibling ? document.querySelector(opts.sibling) : null;
```

Comment:
this was specific to RHR, the stickey header technically needs a sibling. I think.